kv_test.go

```diff
@@ -235,9 +229,7 @@ func TestDelete(t *testing.T) {
 	defer api.AssertExpectations(t)
 	client := pluginapi.NewClient(api)
 
-	api.On("KVSetWithOptions", "1", nil, model.PluginKVSetOptions{
-		EncodeJSON: true,
-	}).Return(true, nil)
+	api.On("KVSetWithOptions", "1", []byte(nil), model.PluginKVSetOptions{}).Return(true, nil)
```

Comment:
I thought using `nil` instead of `[]byte(nil)` should be fine here, but it isn't. Maybe a bug in `testify`?